action_msgs/srv/CancelGoal.srv

```diff
@@ -11,5 +11,23 @@
 # Goal info containing an ID and timestamp
 GoalInfo goal_info
 ---
+# Response codes
+# Indicates the request was accepted.
+# One or more goals have transitioned to the CANCELING state.
+# The goals_canceling list should not be empty.
+int8 RESPONSE_OK=0
+# Indicates the request was rejected.
+# No goals have transitioned to the CANCELING state.
+# The goals_canceling list should be empty.
+int8 RESPONSE_REJECTED=1
+# Indicates the requested goal ID is invalid.
+# This could be because a goal with the ID is not cancelable (e.g. terminated)
+# or there is no goal that matches the ID.
```

Comment:
Are there other reasons (than terminated) for not being cancelable? If yes, maybe they would benefit from separate codes.

Reply:
Not that I can think of. I can make them separate error codes: `UNKNOWN_GOAL_ID` and `GOAL_TERMINATED`

Reply:
29eb7ff